677 Add hybrid programming example for bundle (#678)
Project-MONAI/tutorials: modules/bundle/custom_component/README.md

# Description

> This example mainly shows a typical use case that brings customized python component(like: transform, network, metrics, etc.) in the `train` config.

Reviewer: ```suggestion
This example mainly shows a typical use case that brings customized python components (such as transform, network, metrics) in a configuration-based workflow.
```

Author: updated in the latest commit.

Thanks.

Please note that this example depends on the `spleen_segmentation` bundle example and executes via overriding the config file of it.

## commands example

> Export the customized python code to `PYTHONPATH`:

Reviewer: perhaps comments that "To run the workflow with customized components, `PYTHONPATH` should be revised to include the path to the customized component."

and we should make sure the folder `custom_component/scripts` is a valid python module (it has a `__init__.py` file in the folder)

Author: updated in the latest commit.

Thanks.
